event/event.go

```diff
@@ -152,8 +152,8 @@ func (b *bus) PublishID(ctx context.Context, typ Type, sessionID string, payload
 
 func (b *bus) publish(ctx context.Context, typ Type, sessionID string, payload interface{}) error {
 	log.Debugw("publish", "type", typ, "payload", payload)
-	b.lk.Lock()
-	defer b.lk.Unlock()
+	b.lk.RLock()
```

Comment:
Relaxed these as they weren't a real issue and allow us to actually queue multiple jobs, otherwise we will still hang on every handler run for it to complete for the short term.